### PR TITLE
Coordinate converter plug-in

### PR DESCRIPTION
**Pull request by sindizzy**
A plug-in that converts coordinates from one projection to another.

### Checklist

- [x] I have included examples or tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file
- [x] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

A plug-in that converts coordinates from one projection to another. 
I have the plug-in working with one exception. It seems the source code for plugins all goes in \DotSpatial\Source\Plugins but the obj files get put in \DotSpatial\Source. Not sure what I have to do to change that in my plugin project. I'm Open to suggestions.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 ## V4.0
 
+### Added
```
Comment (jany-tenaj) [inline Changelog.md:6]: Don't put this in 4.0. That was already released. Add it to the Unreleased entry I committed.

**Comment by jany-tenaj** [comment]
Please remove the changes to FormContour.resx. If they are needed create a new pull request. But they shouldn't be needed for your new plugin.

**Comment by sindizzy** [comment]
Ok removed FormContour.resx. I think that was generated when I clicked in the Properties page.
Also updated my changelog on this and all my other PRs.

**Comment by jany-tenaj** [comment]
The build output goes to DotSpatial\Source\bin as it's supposed to. This is done to create a fully functional DemoMap program.

**Pull request merged**